diff --git a/musicbrainzngs/musicbrainz.py b/musicbrainzngs/musicbrainz.py
index beb0a79..bce8df8 100644
--- a/musicbrainzngs/musicbrainz.py
+++ b/musicbrainzngs/musicbrainz.py
@@ -74,9 +74,9 @@ VALID_INCLUDES = {
     'url': RELATION_INCLUDES,
     'discid': [ # Discid should be the same as release
         "artists", "labels", "recordings", "release-groups", "media",
-        "artist-credits", "discids", "isrcs",
+        "artist-credits", "discids", "isrcs", "genres",
         "recording-level-rels", "work-level-rels", "annotation", "aliases"
-    ] + RELATION_INCLUDES,
+    ] + RELATION_INCLUDES + TAG_INCLUDES,
     'isrc': ["artists", "releases", "isrcs"],
     'iswc': ["artists"],
     'collection': ['releases'],
